firmware/Core/Src/littlefs/flash_driver.c

```diff
@@ -220,37 +379,49 @@ FLASH_error_enum_t FLASH_write_disable(SPI_HandleTypeDef *hspi, uint8_t chip_num
             return FLASH_ERR_DEVICE_BUSY_TIMEOUT;
         }
 
-        DEBUG_uart_print_str("DEBUG: status_reg = 0x");
-        DEBUG_uart_print_array_hex(status_reg_buffer, 1);
-        DEBUG_uart_print_str("\n");
+        if (FLASH_enable_hot_path_debug_logs) {
+            DEBUG_uart_print_str("DEBUG: status_reg = 0x");
+            DEBUG_uart_print_array_hex(status_reg_buffer, 1);
```

Comment:
Are we keeping these?

Reply:
`FLASH_enable_hot_path_debug_logs` is 0 by default, so I think it's alright to keep. If you think it's better to remove however I'm okay with that.

Reply:
Please keep :)